client/components/pagination/style.scss

```diff
@@ -48,28 +55,41 @@
 }
 
 .woocommerce-pagination__page-picker {
-	margin-left: $spacing / 2;
-}
+	margin-left: $spacing;
 
-.woocommerce-pagination__page-picker-input {
-	margin-left: $spacing / 2;
-	width: 60px;
+	@include breakpoint( '<782px' ) {
+		margin-top: 14px;
```

Comment:
Looking at the InVision designs, I think that should be `16px` (you can use the variable `$gap`, instead).

Reply:
FYI @joshuatf you should have gotten an invite to Invision in your gmail inbox 🌟

Reply:
@Aljullu Good catch on that and thanks for the `$gap` tip!
@timmyc Received access to InVision!